### PR TITLE
feat: Banner Text theming

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to apply the new [Theming structure](https://github.com/vtex-sites/base.store/pull/407) to the `BannerText` component.

## How does it work?

This PR uses local variables to parameterize the `BannerText` properties, then connects these scoped tokens to the global ones.

### New global tokens:
None.

#### Default properties:
| Local token | Default value/Global token linked |
| - | - | 
| `--fs-banner-text-padding-mobile ` | `var(--fs-spacing-6) 5%` |
| `--fs-banner-text-padding-desktop` | `var(--fs-spacing-9) 15%` |
| `--fs-banner-text-link-min-width` | `11.25rem` |
| `--fs-banner-text-link-margin-top` | `var(--fs-spacing-6)` |
| `--fs-banner-text-title-size` | `var(--fs-text-size-lead)` |
| `--fs-banner-text-title-weight` | `var(--fs-text-weight-bold)` |
| `--fs-banner-text-title-line-height` | `1.2` |

#### Primary:
| Local token | Default value/Global token linked |
| - | - | 
| `--fs-banner-text-primary-title-size` | `var(--fs-text-size-title-page)` |

#### Secondary:
| Local token | Default value/Global token linked |
| - | - | 
| `--fs-banner-text-secondary-title-size` | `var(--fs-text-size-4)` |
| `--fs-banner-text-secondary-caption-size` | `var(--fs-text-size-base)` |
| `--fs-banner-text-secondary-caption-weight` | `1.2` |
| `--fs-banner-text-secondary-caption-line-height` | `1.5` |

#### Main:

| Local token | Default value/Global token linked |
| - | - | 
| `--fs-banner-text-main-bkg-color` | `var(--fs-color-primary-bkg)` |
| `--fs-banner-text-main-text-color` | `var(--fs-color-primary-text)` |

#### Light:

| Local token | Default value/Global token linked |
| - | - | 
| `--fs-banner-text-light-bkg-color` | `var(--fs-color-secondary-bkg-light)` |
| `--fs-banner-text-light-text-color` | `var(--fs-color-secondary-text)` |

#### Accent:

| Local token | Default value/Global token linked |
| - | - | 
| `--fs-banner-text-accent-bkg-color` | `var(--fs-color-highlight-bkg)` |
| `--fs-banner-text-accent-text-color` | `var(--fs-banner-text-light-text-color)` |

## How to test it?

The `BannerText` component should look just as it was before these changes.

Its variants can be tested through Storybook:

- Primary:
Screen Shot 2022-04-13 at 16 28 07

- Secondary:
Screen Shot 2022-04-13 at 16 28 22

## Checklist

- [x] CHANGELOG entry added

### CODE REVIEW

File: src/components/sections/BannerText/banner-text.scss [hunk 1]
```diff
@@ -1,44 +1,100 @@
...
+  --fs-banner-text-padding-mobile              : var(--fs-spacing-6) 0;
+  --fs-banner-text-padding-desktop             : var(--fs-spacing-9) 0;
+
+  --fs-banner-text-bkg-color                   : var(--fs-color-main-4);
+  --fs-banner-text-text-color                  : var(--fs-color-text-inverse);
+
+  --fs-banner-text-link-width                  : 11.25rem;
+  --fs-banner-text-link-padding                : var(--fs-spacing-2) var(--fs-spacing-4);
+  --fs-banner-text-link-margin-mobile          : var(--fs-spacing-5) 0 0 0;
+  --fs-banner-text-link-margin-desktop         : var(--fs-spacing-7) 0 0 0;
```
Comment (renatamottam) [inline src/components/sections/BannerText/banner-text.scss:17]: Shouldn't it be only `margin-top`?
(You should also change it on lines 53 and 57)
```suggestion
  --fs-banner-text-link-margin-top-mobile          : var(--fs-spacing-5);
  --fs-banner-text-link-margin-top-desktop         : var(--fs-spacing-7);
```

File: src/components/sections/BannerText/banner-text.scss [hunk 2]
```diff
@@ -1,44 +1,100 @@
...
+  // Design Tokens for Banner Text
+  // --------------------------------------------------------
+
+  --fs-banner-text-padding-mobile              : var(--fs-spacing-6) 0;
+  --fs-banner-text-padding-desktop             : var(--fs-spacing-9) 0;
+
+  --fs-banner-text-bkg-color                   : var(--fs-color-main-4);
+  --fs-banner-text-text-color                  : var(--fs-color-text-inverse);
+
+  --fs-banner-text-link-width                  : 11.25rem;
```
Comment (renatamottam) [inline src/components/sections/BannerText/banner-text.scss:14]: Maybe we should change this for `min-width`

**Review by renatamottam — COMMENTED** [review]
Nice work, @lucasfp13, we're almost there! I left a few improvement suggestions :)

**Review by renatamottam — APPROVED** [review]
Looks amazing :))))

**Review by filipewl — APPROVED** [review]
LGTM! 

Any chance to add an example to the Storybook?

**Pull request merged**